src/cpp/session/modules/SessionFindTests.cpp

```diff
@@ -246,6 +275,36 @@ TEST_CASE("SessionFind")
       CHECK(match[2].str().compare("1") == 0);
    }
 
+   SECTION("Replace regex with quantifiers")
+   {
+      std::string line("helllooo");
+      const std::string regex("l+o+");
+      const std::string replacement("LO!");
+      Replacer replacer(false);
+      const size_t matchOn = 2;
+      const size_t matchOff = 7;
+      size_t replaceMatchOff;
+
+      replacer.replaceRegex(matchOn, matchOff, regex, replacement, &line, &replaceMatchOff);
+      CHECK(line.compare("heLO!") == 0);
+      CHECK(replaceMatchOff == 4);
+   }
+
+   SECTION("Replace regex with quantifiers")
```

Comment:
```suggestion
   SECTION("Replace regex with quantifiers and backreference")
```
We should use unique section names for these tests so that it's obvious which test is failing when there is an issue.